doc-content/enterprise-only/admin-and-config/kie-server-managing-multiple-containers-proc.adoc

```diff
@@ -1,7 +1,7 @@
 [id='kie-server-managing-multiple-containers-proc']
 = Managing multiple KIE containers
 
-The {KIE_SERVER} allows you to create and provision multiple KIE containers (deployment units). Use the following procedure to create and interact with multiple containers:
+You can create and provision multiple KIE containers (deployment units). Use the following procedure to create and interact with multiple containers:
```

Comment:
Much simpler language here would be:

This procedure shows how to create and interact with multiple KIE containers (deployment units).

Reply:
Adopted with a small change to the wording.